src/sonic-config-engine/tests/sample_output/py3/qos-arista7800r3-48cq2-lc.json

```diff
@@ -497,1155 +497,957 @@
     },
     "QUEUE": {
         "dut-lc3|Asic0|Ethernet0|3": {
```

Comment:
@arlakshm  Can we avoid space change if not code change  ?

Reply:
I fixed the removed the code spaces in the qos_config.j2, which resulted this